src/Widgets/FooterWidget/FooterWidgetComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import { ContentTag, InPlaceEditingOff, Obj, provideComponent } from 'scrivito'
+import { FooterWidget } from './FooterWidgetClass'
+import { isHomepage } from '../../Objs/Homepage/HomepageObjClass'
+
+provideComponent(FooterWidget, () => {
+  const root = Obj.root()
+  if (!isHomepage(root)) return null
+  if (!root.get('layoutShowFooter')) return null
```

Comment:
* [ ] FYI From just reading the descriptions in the UI an editor could assume that the footer widget also works if `layoutShowFooter` is off on the homepage.